### PR TITLE
Update log levels used in com1DFA, plot functions and fileHandlerUtils 

### PR DESCRIPTION
**Pull request by awirb**
* contribute to issue #177

### CODE REVIEW

File: avaframe/out3Plot/outQuickPlot.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ def generatePlot(dataDict, avaName, outDir, cfg, plotDict):
     ax[1].legend()
     fig.savefig(os.path.join(outDir, 'Profiles_%s_%s.%s' % (avaName, simName, outputFormat)))
 
-    log.info('Figures saved to: %s' % outDir)
+    log.debug('Figures saved to: %s' % outDir)
```
Comment (fso42) [inline avaframe/out3Plot/outQuickPlot.py:124]: mh, maybe leave this on level info? This is something the user might look for quite often...

File: avaframe/runCom1DFA.py [hunk 2]
```diff
@@ -24,8 +24,8 @@
 
 # Start logging
 log = logUtils.initiateLogger(avalancheDir, logName)
-log.info('MAIN SCRIPT')
-log.info('Current avalanche: %s', avalancheDir)
+log.debug('MAIN SCRIPT')
+log.debug('Current avalanche: %s', avalancheDir)
```
Comment (fso42) [inline avaframe/runCom1DFA.py:28]: I think these should stay in Info, as this is an important info...

**Review by fso42 — CHANGES_REQUESTED** [review]

**Review by matthiasto — COMMENTED** [review]
Looks good, I agree with Felixs comments.

File: avaframe/out3Plot/outQuickPlot.py (hunk 1 above)
Comment (awirb) [reply avaframe/out3Plot/outQuickPlot.py:124]: Changed to info level

File: avaframe/runCom1DFA.py (hunk 2 above)
Comment (awirb) [reply avaframe/runCom1DFA.py:28]: After loading the configuration in com1DFA there is an info about 'Your current avalanche...' so this is why I removed this here, but changed it back to be info

Comment (fso42) [reply avaframe/runCom1DFA.py]: ahhh, sorry, forgot about that one. Then keep the debug level (since config *has* to be read anyway)

Comment (awirb) [reply avaframe/runCom1DFA.py:28]: ok I have set it back to debug :)

**Review by fso42 — APPROVED** [review]

**Pull request merged**